packages/@divvi/mobile/src/viem/estimateFeesPerGas.test.ts

```diff
@@ -31,101 +27,103 @@ describe(estimateFeesPerGas, () => {
     }
     const fees = await estimateFeesPerGas(client as any)
     expect(fees).toEqual({
-      maxFeePerGas: BigInt(110),
+      maxFeePerGas: BigInt(70), // baseFeePerGas * 1.2 (default multiplier in viem)
```

Comment:
If baseFeePerGas is 50 I believe it will be 60?
```suggestion
      maxFeePerGas: BigInt(60), // baseFeePerGas * 1.2 (default multiplier in viem)
```

Reply:
Ah yes sorry, I forgot to add this in the comment  `+ maxPriorityFeePerGas`.